protostar/cheatable_starknet/callable_hint_locals/call_hint_local.py

```diff
@@ -46,6 +53,13 @@ async def _call(
         calldata: Optional[CairoOrPythonData] = None,
     ) -> CairoData:
         try:
+            self._expect_call_controller.remove_expected_call(
+                ExpectedCall(
+                    address=contract_address,
+                    fn_selector=entry_point_selector,
+                    calldata=calldata if calldata else [],
```

Comment:
```suggestion
                    calldata=calldata or [],
```

Reply:
[change remove expected call from external to static](https://github.com/software-mansion/protostar/pull/1481/commits/1179d77e4e82e4d969ac27da7712c09d39001975)